README.md

```diff
@@ -5,38 +5,38 @@
 
 ## Overview
 
-The baremetal service is essentially a cut down version of the [Kubernetes service](https://github.com/unikorn-cloud/kubernetes) that provisions its own baremetal servers using hardware abstraction prvoided by the [Region service](https://github.com/unikorn-cloud/region).
+The baremetal service is essentially a cut down version of the [Kubernetes service](https://github.com/unikorn-cloud/kubernetes) that provisions its own baremetal servers using hardware abstraction provided by the [Region service](https://github.com/unikorn-cloud/region).
 
-Where possible, as the Baremetal service is very similar to the Kubernetes service, we must maintain type and API parity ot ease creation of UX tools and services.
+Where possible, as the Baremetal service is very similar to the Kubernetes service, we must maintain type and API parity to ease creation of UX tools and services.
 
 ## Installation
 
 ### Unikorn Prerequisites
 
-The use the Baremetal service you first need to install:
+To use the Baremetal service you first need to install:
 
-* [The identity service](https://github.com/unikorn-cloud/identity) to provide API authentication and authorization.
-* [The region service](https://github.com/unikorn-cloud/region) to provide provider agnostic cloud services (e.g. images, flavors and identity management).
+- [The identity service](https://github.com/unikorn-cloud/identity) to provide API authentication and authorization.
```

Comment:
Again, this is introducing inconsistency across everything.  `*` is perfectly valid unordered list markdown and used everywhere else.  There will be times when some magical regular expression may be required across all the things and your snowflake will cause it to break/not apply unless you're actually aware (sad enough to know) of every possible permutation.